src/index.tsx

```diff
@@ -278,6 +299,12 @@ const ReactImagePickerEditor = memo(({ config = {}, imageSrcProp = '', color = '
 
   }
 
+  useImperativeHandle(ref, () => ({
+    onRemove,
+    onOpenEditPanel,
+    handleFileSelect,
+  }));
```

Comment:
2 cái `on` chuyển thành props thì hợp lí hơn là ref. Ref k có giá trị gì ở đây lắm 

```suggestion
  useImperativeHandle(ref, () => ({
    handleFileSelect,
  }));
```

Reply:
@tienshake ?